tools/accuracy_checker/configs/midasnet.yml

```diff
@@ -0,0 +1,21 @@
+models:
+  - name: midasnet
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model: public/midasnet/FP32/midasnet.xml
+        weights: public/midasnet/FP32/midasnet.bin
+        adapter: mono_depth
```

Comment:
could you please also provide launcher for FP16 model here?
We need it for validation. It is very simillar to FP32:
```yml
      - framework: dlsdk
        tags:
          - FP16
        model: public/midasnet/FP16/midasnet.xml
        weights: public/midasnet/FP16/midasnet.bin
        adapter: mono_depth

Reply:
added